installation/install.sh

```diff
@@ -120,24 +120,35 @@ echo "web_server_domain=$web_server_domain" > $app_home/host.env
 echo "web_server_port=$web_server_port" >> $app_home/host.env
 
 #setup .env and server_pm2.json
+export LC_CTYPE=C
+oauth_secret=`cat /dev/urandom|tr -dc '[:alpha:]'|fold -w ${1:-20}|head -n 1`
+sendmail_key=`cat /dev/urandom|tr -dc '[:alpha:]'|fold -w ${1:-20}|head -n 1`
+jwt_key=`cat /dev/urandom|tr -dc '[:alpha:]'|fold -w ${1:-20}|head -n 1`
```

Comment:
Will you add a comment saying what the command does?

Reply:
I tried running the command on my Mac and it didn't run.

<img width="547" alt="image" src="https://github.com/microbiomedata/nmdc-edge/assets/134325062/f47de4aa-66d0-4df1-8e93-41d552789f34">

That is in zsh 5.9 running on a MacBook Pro M1 (MacOS Ventura 13.6).

Will you also document the system requirements for the command?

Reply:
'export LC_CTYPE=C' is used to fix the 'tr: Illegal byte sequence' error on MacOS.

Reply:
I tested the shell script on my MacOS and nmdc-edge Ubuntu.

Reply:
Thanks!